inkcpp/value.h

```diff
@@ -96,10 +96,18 @@ namespace ink
 
 				// == Getters ==
 				int as_int() const { return _first.integer_value; }
+				int* as_int_ptr() { return &_first.integer_value; }
+				const int* as_int_ptr() const { return &_first.integer_value; }
 				float as_float() const { return _first.float_value; }
+				float* as_float_ptr() { return &_first.float_value; }
+				const float* as_float_ptr() const { return &_first.float_value; }
 				uint32_t as_divert() const { return _first.uint_value; }
 				uint32_t as_thread_id() const { return _first.uint_value; }
-				// TODO: String access?
+				uint32_t* as_uint_ptr() { return &_first.uint_value; }
+				const uint32_t* as_uint_ptr() const { return &_first.uint_value; }
+				// FIXME: we should compress the value before
```

Comment:
Do you still need this comment? Looks like you're calling finalize string in both these cases now.